src/guiFormSpecMenu.h

```diff
@@ -400,6 +398,7 @@ class GUIFormSpecMenu : public GUIModalMenu
 	std::vector<ImageDrawSpec> m_images;
 	std::vector<ImageDrawSpec> m_itemimages;
 	std::vector<BoxDrawSpec> m_boxes;
+	std::map<std::string, bool> field_close_on_enter;
```

Comment:
please use UNORDERED_MAP macro if map element order is not important to improve performance on >=C++11 builds